src/components/ProfilePage.tsx

```diff
@@ -153,6 +161,33 @@ export const ProfilePage = (props: ProfilePageProps) => {
         </>
     ) 
 
+    const onCropComplete = (croppedArea: any, croppedAreaPixels: any) => {
+		setCroppedArea(croppedAreaPixels);
+	};
+
+    const onCrop = async() => {
+        const file: any = await getCroppedImg(image!, croppedArea!)
+        try{
+            setLoadImg(true)
```

Comment:
Probably only need to call `setLoadImg` once in this function. I'm not sure where that should be, but intuitively I think after the image has been uploaded to Synapse, and shouldn't occur if that upload results in a thrown error?